Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM debian:bullseye as assets
```

Comment:
@alephnull @ermirizio does this Dockerfile used anywhere in the CI?
This is our main one, right https://github.com/TykTechnologies/tyk/blob/master/ci/Dockerfile.std ?

Reply:
@titpetric I see that it probably some "development" Docker image. Do you or anyone from the team use it?

Reply:
Not in CI, local dev only. I use it regularly, people have used it, it's being updated according to dev needs along with docker compose.

Reply:
Dockerfile.std is what the official images are based on. Every repo had one and it is used in CI as well.